griffon/output.py

```diff
@@ -216,6 +216,14 @@ def generate_normalised_results(
                                     normalised_results.append(c)
                                 if item.get("type") == output_type_filter:
                                     normalised_results.append(c)
+                            else:
+                                logger.debug(f'.griffon blocked component {item["name"]}')
+                        else:
+                            logger.debug(f'prod def blocked component {item["name"]}')
+                    else:
+                        logger.debug(f'.griffonrc product streams blocked component {item["name"]}')
+                else:
+                    logger.debug(f'product stream not active {ps}')
```

Comment:
Nitpick: maybe in the future, we would like to restructure this for better readability by flipping the conditions to something like this:
```python
if condition1:
    logger.debug(...)
    continue

if condition2:
    logger.debug(...)
    continue
    
...

do_something_with_data()
```

Reply:
removed debug statements (only needed them temporarily)